bolt/internal/internal.go

```diff
@@ -747,6 +747,87 @@ func UnmarshalConfigPB(data []byte, c *Config) error {
 	return proto.Unmarshal(data, c)
 }
 
+// MarshalOrganizationConfig encodes a config to binary protobuf format.
+func MarshalOrganizationConfig(c *chronograf.OrganizationConfig) ([]byte, error) {
+	columns := make([]*LogViewerColumn, len(c.LogViewer.Columns))
+
+	for i, column := range c.LogViewer.Columns {
+		encodings := make([]*ColumnEncoding, len(column.Encodings))
+
+		for j, e := range column.Encodings {
+			encodings[j] = &ColumnEncoding{
+				Type:  e.Type,
+				Value: e.Value,
+				Name:  e.Name,
+			}
+		}
+
+		columns[i] = &LogViewerColumn{
+			Name:      column.Name,
+			Position:  column.Position,
+			Encodings: encodings,
+		}
+	}
+
+	return MarshalOrganizationConfigPB(&OrganizationConfig{
+		OrganizationID: c.OrganizationID,
+		LogViewer: &LogViewerConfig{
+			Columns: columns,
+		},
+	})
+}
+
+// MarshalOrganizationConfigPB encodes a config to binary protobuf format.
+func MarshalOrganizationConfigPB(c *OrganizationConfig) ([]byte, error) {
+	return proto.Marshal(c)
+}
+
+// UnmarshalOrganizationConfig decodes a config from binary protobuf data.
+func UnmarshalOrganizationConfig(data []byte, c *chronograf.OrganizationConfig) error {
+	var pb OrganizationConfig
+
+	if err := UnmarshalOrganizationConfigPB(data, &pb); err != nil {
+		return err
+	}
+
+	if pb.OrganizationID == "" {
```

Comment:
`UnmarshalOrganizationConfig` shouldn't do this kind of validation

Reply:
Previous work on UnmarshalConfig includes this kind of guard for the `Auth` field. Happy to remove it if you think it's not necessary. Do you think we should include this kind of guard elsewhere?

Reply:
I'd keep it for things that would panic if they weren't set. Looking here, I'd guess that would be `pb.LogViewer`. Everything else should be good.

> Do you think we should include this kind of guard elsewhere?
This validation should be done in the one or both of the http handler and the implementers of `chronograf.OrganizationConfigStore`.

Reply:
👍